### ISSUE TITLE
Notable infobox not properly handled

### ISSUE THREAD

**Issue by MaZDuR** [Pegoth/UniversalClose#11]
I noticed this a little while ago (1.2.3) whenever you are in a village/town and look at a quest/issue available from a notable, pressing tab will exit the village/town instead of closing the infobox.

What box am I talking about? This one:
![image](https://user-images.githubusercontent.com/1400709/80318328-662a7b80-87d7-11ea-910e-b78528b52afe.png)

**Comment by Pegoth** [comment]
Does that also happen without the mod? I don't think I even look for this window / village town menu.

**Comment by MaZDuR** [comment]
Yes, default is to do nothing, mod doesn't add tab to close on these and instead exits you from the settlement
On Apr 26, 2020 4:06 PM, Pegoth <notifications@github.com> wrote:
Does that also happen without the mod? I don't think I even look for this window / village town menu.
